style.css

```diff
@@ -35,64 +35,135 @@ body {
   z-index: 1;
 }
 
-/* #menu-option {
-  z-index: 10;
-} */
-
-/* .nav-bar_desktop {
-  display: none;
-  position: absolute;
-  height: 100%;
-  width: 100%;
-  flex-direction: column;
-  background-color: #fff;
-  z-index: 10;
-} */
-
 .hidden {
   display: none;
 }
 
-/* .nav-bar_desktop ul {
+.nav-bar img {
+  height: 20px;
+}
+
+/* POPUP-MOBILE START HERE */
+
+ul {
   list-style: none;
-  color:#3a4a42;
-  padding-left: 24px;
-  padding-top: 24px;
-} */
+}
+
+.popup {
+  width: 100%;
+  height: 100%;
+  background-color: #c1c7d0;
+  display: none;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  overflow: auto;
+  backdrop-filter: blur(8px);
+  transition: top 3s ease-in-out;
+}
+
+.popup-content {
+  width: 90%;
+  margin: 30px auto;
+  height: auto;
+  background-color: #fff;
+  justify-content: center;
+  top: 60px;
+  padding: 5px 20px;
+  border-radius: 8px;
+  text-align: center;
+}
 
-/* .nav-bar_desktop li {
-  margin-bottom: 25px;
-} */
+.close-btn {
+  float: right;
+  font-size: 20px;
+  color: #c1c7d0;
+  cursor: pointer;
+}
+
+.pop-img {
+  width: 100%;
+  height: 100%;
+}
 
-/* .close {
+.pop-heading {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
   margin-top: 20px;
-  align-self: flex-end;
-  margin-right: 20px;
 }
-.x-mark {
-  height: 15px;
-  width: 15px;
-  color:#3a4a42;
-  z-index: 10;
-} */
-
-/* .nav-bar_desktop a {
-  text-decoration: none;
-  color:#3a4a42;
-  font-family: 'Inter';
-  font-weight: 600;
+
+.pop-heading h3 {
+  font-family: var(--font-family);
+  color: #172b4d;
+  font-style: normal;
+  font-weight: 700;
   font-size: 32px;
   line-height: 44px;
+  display: block;
+  text-align: left;
+}
+
+.pop-button {
+  display: none;
+}
+
+.pop-button-mob {
   display: flex;
-  align-items: center;
-  padding-left: 16px;
-  z-index: 10;
-} */
+}
 
-.nav-bar img {
-  height: 20px;
+.button-info-mob {
+  padding: 15px;
+  font-size: 15px;
+  font-weight: 600;
+  border-radius: 8px;
+  border: none;
+  color: #fff;
+  background-color: #36b37e;
+  margin-right: 20px;
+}
+
+.button-info img {
+  margin-top: 10px;
+}
+
+.pop-skills {
+  display: none;
+}
+
+.pop-skills-mob {
+  margin-top: 20px;
```

Comment:
- [ ] Kindly ensure that the text in the buttons is centralized and that the buttons are the same height as in the given design

### Your design
![Screenshot 2023-04-22 at 23 38 48 (2)](https://user-images.githubusercontent.com/64774936/233810770-6bca4b9f-ea51-43ae-b388-541c38251cb5.png)

### Given design
![Screenshot 2023-04-22 at 23 38 53 (2)](https://user-images.githubusercontent.com/64774936/233810775-b0a1a05d-2ddb-4a5d-b9b1-f679185eb1b7.png)